configs/_base_/datasets/waymoD3-mono3d-3class.py

```diff
@@ -0,0 +1,150 @@
+# dataset settings
+# D3 in the config name means the whole dataset is divided into 3 folds
+# We only use one fold for efficient experiments
+dataset_type = 'WaymoDataset'
+data_root = 'data/waymo/kitti_format/'
+class_names = ['Car', 'Pedestrian', 'Cyclist']
+input_modality = dict(use_lidar=False, use_camera=True)
+file_client_args = dict(backend='disk')
+# Uncomment the following if use ceph or other file clients.
+# See https://mmcv.readthedocs.io/en/latest/api.html#mmcv.fileio.FileClient
+# for more details.
+train_pipeline = [
+    dict(type='LoadImageFromFileMono3D', file_client_args=file_client_args),
```

Comment:
You can delete all the `file_client_args=file_client_args` in configs, since we use `replace_ceph_backend` for unified ceph support.

Reply:
remove this